substrate/mmap

```diff
@@ -184,12 +224,14 @@ namespace substrate
 #endif
 		mmap_t(const mmap_t &) = delete;
 		mmap_t &operator =(const mmap_t &) = delete;
+		mmap_t(mmap_t &&) noexcept = default;
+		mmap_t& operator=(mmap_t &&) noexcept = default;
 
 		void swap(mmap_t &map) noexcept
```

Comment:
Same as fd_t, this type's move and swapping requirements may be completely handled by the compiler.

Reply:
Same as with fd_t, will let you know if someone replies.